pkg/command/service/common.go

```diff
@@ -93,6 +100,75 @@ func GetNamespaces(ctx context.Context, params *pkg.PerfParams, namespace, names
 	return nsNameList, nil
 }
 
+// getServices gets existed services by svc or (svc-prefix, svcRange)
+func getServices(ctx context.Context, servingClient servingv1client.ServingV1Interface, nsNameList []string, service string, svcPrefix string, svcRange string) ([]ServicesToScale, error) {
```

Comment:
Since the old version of this was defined as:

```go
func getServices(ctx context.Context, servingClient servingv1client.ServingV1Interface, nsNameList []string, svcPrefix string)
```

should we append the new variables to the end (rather than inserting `service` before `svcPrefix`)?

Reply:
Good idea. will address